README.md

```diff
@@ -1 +1,31 @@
 # bedrock-ledger-test-cli
+
+## Setup:
+
+  - You need to have a [web ledger node](https://github.com/digitalbazaar/bedrock-ledger-app-template/tree/initial) already running somewhere.
```

Comment:
The URL you have linked to here is not going to be valid after that gets merged to master.  Maybe take off the branch info and link just to the repo (even though there's nothing on master yet).

Reply:
as much as I do hate to say this I do think this one is relative as linking to an actually working branch is probably better than master which does not contain working code. If a developer runs into an empty branch I think they can infer which one to use easier than say master. change made though.

Reply:
on the other hand I guess master is better in the long term. way to think about our future.